src/ui/widgets/config/forms/NavigationBarConfigForm.js

```diff
@@ -14,12 +14,20 @@ import AppTourContainer from 'ui/app-tour/AppTourContainer';
 import { APP_TOUR_STARTED } from 'state/app-tour/const';
 
 export default class NavigationBarConfigForm extends PureComponent {
+  constructor() {
+    super();
+    this.state = {
+      isClonePage: false,
```

Comment:
I'm not a fan of those config flags. 
On top of that, I `WidgetForm` already has info about "mode" which could be new | edit | clone.
You could stick to that solution. Not the best possible one, but cleaner than having scattered flags from different sources of truth.

Reply:
this page is only loaded for clone mode, there is a handler in WidgetForm that ensure it will be only loaded for this mode:
```
    const NativeWidgetConfigForm = parentWidget
      && mode === MODE_CLONE
      && getAppBuilderWidgetForm(parentWidget, true);
```

also, this page is loaded dynamically by `getAppBuilderWidgetForm` so I couldn't pass down the `mode` prop to this component, that's why I'm getting it from the URL

Reply:
just checked @ichalagashvili and looks like it is a better solution!
thanks!